index.html

```diff
@@ -0,0 +1,333 @@
+                    <img class="home-testimonial__image" src="/img/bearded-man-in-white-tshirt.png" alt="">
+                    <div class="home-testimonial__profile">
+                        <p class="home-testimonial__profile-name">Sam Preston
+                        </p>
+                        <p class="home-testimonial__profile-professional">CTO</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="component">
+            <div class="component__content">
+                <h1 class="component__title">Are you ready to grow your <br> business with us?</h1>
+                <p class="component__paragraph">Lorem ipsum dolor sit
+                    amet, consetetur
+                    sadipscing elitr, sed diam nonumy.</p>
+                <button class="component__button">View
+                    Pricing</button>
+            </div>
+        </section>
+
+
+
+    <footer class="footer">
+        <div class="footer__cols">
+            <div class="footer__column">
+                <p class="footer__title">Company</p>
+                <ul class="footer__links">
+                    <li class="footer__item"><a href="#" class="footer__link">About Us</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Why Choose us</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Pricing</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Testimonial</a></li>
+                </ul>
+            </div>
+            <div class="footer__column">
+                <p class="footer__title">Resources</p>
+                <ul class="footer__links">
+                    <li class="footer__item"><a href="#" class="footer__link">Privacy Policy</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Terms and Condition</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Blog</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Contact Us</a></li>
+                </ul>
+            </div>
+            <div class="footer__column">
+                <p class="footer__title">Product</p>
+                <ul class="footer__links">
+                    <li class="footer__item"><a href="#" class="footer__link">Project managment</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Time tracker</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Time schedule</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Lead generate</a></li>
+                    <li class="footer__item"><a href="#" class="footer__link">Remote Collaboration</a></li>
+                </ul>
+            </div>
+            <div class="footer__column-logo">
```

Comment:
## improve the name
`footer__column-logo` improve this name. find name that makes sense